### PR TITLE
Webdriverio POC draft - DNM

### PR DESCRIPTION
**Pull request by siggerzz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,10 +11,12 @@
     "storybook:build": "lerna run storybook:build",
     "storybook:serve": "lerna run storybook:serve",
     "storybook:deploy": "storybook-to-ghpages --script storybook:build",
-    "test": "cross-env-shell \"lerna run $LERNA_ARGS test --stream\""
+    "test": "cross-env-shell \"lerna run $LERNA_ARGS test --stream\"",
+    "test-component:chrome": "lerna run test-component:chrome"
   },
   "dependencies": {
-    "core-js": "^3.6.4"
+    "core-js": "^3.6.4",
```
Comment (ashleynolan) [inline package.json:18]: Can you get rid of the `^` on these dependencies?

File: packages/f-header/package.json [hunk 2]
```diff
@@ -28,10 +28,14 @@
...
     "build": "vue-cli-service build --target lib --name f-header ./src/index.js",
-    "demo": "vue serve --open ./src/demo/Index.vue",
+    "demo": "vue serve --port 8080 ./src/demo/Index.vue",
     "lint": "vue-cli-service lint --no-fix",
     "lint:fix": "yarn lint --fix",
-    "test": "vue-cli-service test:unit"
+    "test": "vue-cli-service test:unit",
+    "test-component:chrome": "run-p --race demo webdriver:delay:chrome",
+    "test:wait-for-server": "node ../../test/infrastructure/healthcheck.js 8080",
+    "webdriver:delay:chrome": "yarn test:wait-for-server && yarn webdriver:start:chrome",
```
Comment (ashleynolan) [inline packages/f-header/package.json:37]: Can you add `script-info` blocks for these to explain what they do?

File: test/infrastructure/healthcheck.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+const axios = require('axios');
+
+let attempts = 0;
+let intervalId;
+
+const interval = setInterval(async () => {
```
Comment (ashleynolan) [inline test/infrastructure/healthcheck.js:7]: Can you add JSDoc comments to any new functions? (we're trying to do this for all JS functions going forward)

File: package.json (hunk 1 above)
Comment (DamianMullins) [inline package.json:18]: Does this need to be a `dependency`? We usually don't want `core-js` shipped with all th packages as it is quite large.

File: package.json [hunk 4]
```diff
@@ -11,10 +11,12 @@
...
+    "global": "^4.4.0"
```
Comment (DamianMullins) [inline package.json:19]: Where is this used?

**Pull request closed without merging**